src/main/java/net/liquidpineapple/pang/screens/LevelIterator.java

```diff
@@ -83,7 +84,7 @@ public int compare(LevelScreen level1, LevelScreen level2) {
   private void loadArrayIde(final String directory) {
     //get the files from the directory
     File[] files = new File(Application.class.getResource("/" + directory).getFile()).listFiles();
-    Pattern pattern = Pattern.compile("levels\\\\(single|multi)player\\\\level\\d*.xml");
+    Pattern pattern = Pattern.compile("levels\\\\(single|multi|custom)player\\\\level\\d*.xml");
```

Comment:
Yeah I will change the regex to match the singleplayer/multiplayer/custom format. The result of the regex (stored in the group) will be the path to the different levels.